pypeit/find_objects.py

```diff
@@ -312,33 +312,41 @@ def run(self, std_trace=None, show_peaks=False):
         self.tilts = self.waveTilts.fit2tiltimg(self.slitmask, flexure=tilt_flexure_shift)
         #
 
-        # First pass object finding
+        # Check if the user wants to use a pre-defined sky regions file.
+        skymask0, usersky = self.load_skyregions(None)
+        # JFH Perform a first pass sky-subtraction without masking any objects
+        # TODO: DEIMOS box slits are not being sky-subtracted. Not sure why
```

Comment:
remove if clarified from chat with DP

Reply:
Yes, DEIMOS align boxes are not sky subtracted because we only need them to trace the offsets and we don't want the global_sky calculation to fail.

Reply:
Removed.